### ISSUE TITLE
multi-contigs

### ISSUE THREAD

**Issue by frihaka** [linsalrob/PhiSpy#50]
Hi,

thanks for sharing this tool.
I had expected the method to work as well with a multi-contig input file (cf Phaster). Indeed, it clearly recognizes the number of relevant (>5kb) contigs in the input file (e.g "Processing 2 contigs"). But it seems that it scans only the 1st contig...or did I miss some parameter/twick?

Thanks for your help,

H.

**Comment by frihaka** [comment]
I found the issue - PhiSpy does simply not find a hit (even though there should be one on each contig), hence no report for that one.

**Comment by linsalrob** [comment]
Did you use a hmm search against (e.g.) pvogs or phrogs?

**Comment by frihaka** [comment]
Hi,

I have tried vogs/pvogs, with no more success. But phrogs has worked (i.e given expected results with no false negative anymore).

Thanks for your response, was very helpful.


ps: Maybe I should open another threads for this - since I was running different hmms on the same input data, I wanted to try the --skip_search flag in combination with --phmms. This threw an error: this parameter does not seem to exist in the PhiSpy.py version I have just installed (python3 -m pip install --user PhiSpy - on Ubuntu20). Or does the order of the options/flag matter?
